gaj23/face-it
# Ch feature type component

#### Purpose: 
> Check all that apply.
- [ X] Feature
- [X ] Bug Fix 
- [ X] Clean Up

### What does this PR do? 
> Clearly and concisely describe changes made. 
We split up the Category component into three different components (Eco, Vegan and AllergenFriendly) in order to display the product types for each category. We decided to split it up in order to ensure the data would display correctly. We also went ahead and styled each page with conditional rendering and using images to showcase the product types. We also cleaned up the class names and changed them to camelCase to be consistent. 


### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.
We encountered a major problem with the Category Component when trying to make it dynamic. We ended up splitting up the Category component into three smaller components and that way our Route links stayed the same and nothing ended up broken.


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?
Reviewer should start in the Components folder, and go into the Category folder and review all of the components in the Category folder. Reviewer should also go into the Type component to review the Type component.


### How should this be tested?
> What steps need to be taken to confirm functionality? 
NPM Start and run localhost:3000 in the browser.

### Steps To Take
> Any questions and possible steps moving forward.



#### Checklist:
- [X ] I have performed a self-review of my own code
- [X ] My code follows the style guidelines of this project
